[02-Oct-2022 12:36:07 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(55): Core\Router->dispatch('HNAP1/')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:13:56 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:14:15 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:14:29 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:14:29 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:14:41 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:16:31 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:16:45 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:20:54 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:33:34 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:33:53 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:41:41 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:41:46 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:41:47 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:41:49 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:43:27 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:45:18 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:47:08 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:48:29 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:58:59 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:59:00 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:59:02 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:59:02 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 14:59:04 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:05:22 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:05:23 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:05:23 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:05:24 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:05:26 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:05:26 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:05:28 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:13:14 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:13:19 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:13:23 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:13:23 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:13:25 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:14:54 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:14:57 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:14:57 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:14:59 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('expenses/getExp...')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:01 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main/menu')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:02 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main/menu')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:04 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('settings/index')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:05 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main/menu')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:06 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main/incomes')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:09 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:16 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:17 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:21 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:22 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('favicon.ico')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 15:27:23 Europe/Berlin] Uncaught exception: 'Exception' with message 'No route matched.'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main')
#1 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php' on line 130
[02-Oct-2022 16:59:43 Europe/Berlin] Uncaught exception: 'Error' with message 'Class "App\Controllers\Expense" not found'
Stack trace: #0 [internal function]: App\Controllers\Main->expAction()
#1 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#2 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('exp', Array)
#3 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(57): Core\Router->dispatch('main/exp')
#4 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Main.php' on line 50
[02-Oct-2022 17:26:24 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Array to string conversion'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Array to string...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('18')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:29:16 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('18')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:29:18 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('17')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:29:25 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('18')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:32:11 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('19')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:32:14 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('17')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:32:15 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('18')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:32:16 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('20')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:34:31 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('18')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
[02-Oct-2022 17:34:50 Europe/Berlin] Uncaught exception: 'ErrorException' with message 'Undefined array key "userLimit"'
Stack trace: #0 C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php(195): Core\Error::errorHandler(2, 'Undefined array...', 'C:\\xampp\\htdocs...', 195)
#1 C:\xampp\htdocs\php-mvc-personalBudget\App\Controllers\Expenses.php(43): App\Models\Expense::getLimit('17')
#2 [internal function]: App\Controllers\Expenses->getCategoryLimitAction()
#3 C:\xampp\htdocs\php-mvc-personalBudget\Core\Controller.php(50): call_user_func_array(Array, Array)
#4 C:\xampp\htdocs\php-mvc-personalBudget\Core\Router.php(121): Core\Controller->__call('getCategoryLimi...', Array)
#5 C:\xampp\htdocs\php-mvc-personalBudget\public\index.php(62): Core\Router->dispatch('expenses/getCat...')
#6 {main}
Thrown in 'C:\xampp\htdocs\php-mvc-personalBudget\App\Models\Expense.php' on line 195
